starlette/config.py

```diff
@@ -12,8 +12,8 @@ class EnvironError(Exception):
     pass
 
 
-class Environ(MutableMapping):
-    def __init__(self, environ: typing.MutableMapping = os.environ):
+class Environ(MutableMapping[typing.Any, typing.Any]):
```

Comment:
Should this be `Any` for both key and value types? `Config` expects a `typing.Mapping[str, str]` for its `environ` argument

Reply:
I'm not too familiar with this part of Starlette, but env variables (e.g. `os.environ`) are always string keys and string values

Reply:
I went for `Any` as dunder methods were typed with `Any`, but I guess it makes more sense to have `str` instead